### PR TITLE
Add solution of chatglm2-6b cannot set attribute to Readme

### PR DESCRIPTION
**Pull request by wgzintel**

### CODE REVIEW

File: llm_bench/python/README.md [hunk 1]
```diff
@@ -110,3 +110,18 @@ Add the option `--torch_compile_backend` with the desired backend: `pytorch` or
 ```bash
 python ./benchmark.py -m models/llama-2-7b-chat/pytorch -d CPU --torch_compile_backend openvino
 ```
+
```
Comment (peterchen-intel) [inline llm_bench/python/README.md:113]:
```suggestion
# Notes
```

File: llm_bench/python/README.md [hunk 2]
```diff
@@ -110,3 +110,18 @@ Add the option `--torch_compile_backend` with the desired backend: `pytorch` or
...
+## Fix tokenizer initialization issue AttributeError of chatglm2-6b: can't set attribute
```
Comment (peterchen-intel) [inline llm_bench/python/README.md:114]:
```suggestion
## chatglm2-6b - AttributeError: can't set attribute
```

File: llm_bench/python/README.md [hunk 3]
```diff
@@ -110,3 +110,18 @@ Add the option `--torch_compile_backend` with the desired backend: `pytorch` or
...
+When running the chatglm2-6b downloaded from hugginface and converted to openvino IR, the following error may occur：
+```bash
+AttributeError: can't set attribute
+```
```
Comment (peterchen-intel) [inline llm_bench/python/README.md:118]: Reproduced with https://huggingface.co/THUDM/chatglm2-6b 7fabe56db91e085c9c027f56f1c654d137bdba40

File: llm_bench/python/README.md [hunk 4]
```diff
@@ -110,3 +110,18 @@ Add the option `--torch_compile_backend` with the desired backend: `pytorch` or
...
+The solution is to modify `tokenization_chatglm.py` in the openvino IR directory and insert the following code in `line 73`
```
Comment (peterchen-intel) [inline llm_bench/python/README.md:119]:
```suggestion
As on https://huggingface.co/THUDM/chatglm2-6b/discussions/99<br>
Solution: update `tokenization_chatglm.py` as following:
```

File: llm_bench/python/README.md (hunk 3 above)
Comment (wgzintel) [reply llm_bench/python/README.md:118]: updated

**Review by peterchen-intel — APPROVED** [review]

File: llm_bench/python/README.md [hunk 5]
```diff
@@ -110,3 +110,23 @@ Add the option `--torch_compile_backend` with the desired backend: `pytorch` or
...
+{
+          self.vocab_file = vocab_file
+          self.tokenizer = SPTokenizer(vocab_file)
+ +        kwargs.pop("eos_token", None)
+ +        kwargs.pop("pad_token", None)
+ +        kwargs.pop("unk_token", None)
+          self.special_tokens = {
+              "<bos>": self.tokenizer.bos_id,
+              "<eos>": self.tokenizer.eos_id,
+}
```
Comment (peterchen-intel) [inline llm_bench/python/README.md:132]:
```suggestion
```Python
          self.vocab_file = vocab_file
          self.tokenizer = SPTokenizer(vocab_file)
 +        kwargs.pop("eos_token", None)
 +        kwargs.pop("pad_token", None)
 +        kwargs.pop("unk_token", None)
          self.special_tokens = {
              "<bos>": self.tokenizer.bos_id,
              "<eos>": self.tokenizer.eos_id,
```

File: llm_bench/python/README.md [hunk 6]
```diff
@@ -110,3 +110,21 @@ Add the option `--torch_compile_backend` with the desired backend: `pytorch` or
...
+Solution: update `tokenization_chatglm.py` as following: <br />
+```Python
+          self.vocab_file = vocab_file
+          self.tokenizer = SPTokenizer(vocab_file)
+ +        kwargs.pop("eos_token", None)
+ +        kwargs.pop("pad_token", None)
+ +        kwargs.pop("unk_token", None)
+          self.special_tokens = {
+              "<bos>": self.tokenizer.bos_id,
+              "<eos>": self.tokenizer.eos_id,
```
Comment (peterchen-intel) [inline llm_bench/python/README.md:130]:
```suggestion
              "<eos>": self.tokenizer.eos_id,
              
> The solution works for chatglm3-6b as well.
```

File: llm_bench/python/README.md [hunk 7]
```diff
@@ -110,3 +110,21 @@ Add the option `--torch_compile_backend` with the desired backend: `pytorch` or
 ```bash
 python ./benchmark.py -m models/llama-2-7b-chat/pytorch -d CPU --torch_compile_backend openvino
 ```
+# Notes
+## chatglm2-6b - AttributeError: can't set attribute
+When running the chatglm2-6b downloaded from hugginface and converted to openvino IR, the following error may occur：
```
Comment (peterchen-intel) [inline llm_bench/python/README.md:115]:
```suggestion
Download chatglm2-6b from hugginface, convert to OpenVINO IR files and run with benchmark.py, the following error may occur：
```

**Review by peterchen-intel — APPROVED** [review]

File: llm_bench/python/README.md [hunk 8]
```diff
@@ -110,3 +110,23 @@ Add the option `--torch_compile_backend` with the desired backend: `pytorch` or
...
+          self.vocab_file = vocab_file
+          self.tokenizer = SPTokenizer(vocab_file)
+ +        kwargs.pop("eos_token", None)
+ +        kwargs.pop("pad_token", None)
+ +        kwargs.pop("unk_token", None)
+          self.special_tokens = {
+              "<bos>": self.tokenizer.bos_id,
+              "<eos>": self.tokenizer.eos_id,
+              
+> The solution works for chatglm3-6b as well.
```
Comment (peterchen-intel) [inline llm_bench/python/README.md:132]:
```suggestion

> The solution works for chatglm3-6b as well.
```

**Pull request merged**